.skillsaw/plugindocs_rule.py

```
@@ -40,95 +81,98 @@ def check(self, context: RepositoryContext) -> List[RuleViolation]:
+        if not violations:
+            return results
+
+        plugins_md_path = context.root_path / "PLUGINS.md"
+        data_json_path = context.root_path / "docs" / "data.json"
+
+        try:
+            (
+                original_plugins_md, generated_plugins_md,
+                original_data_json, generated_data_json,
+            ) = self._run_generators(context)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)